app/templates/client/dev/components/todo/todo.html

```diff
@@ -1,18 +1,13 @@
 <div class="container">
-
   <div class="todo-panel transition">
-
-    <form name="TodoForm"
-          ng-submit="todo.createTodo(todo.todo)" >
-
+    <form name="TodoForm" ng-submit="todo.createTodo(todo.todo)">
```

Comment:
Could we keep ```ng-submit``` under name? I think it's easier to spot.

If you feel like the ```div``` under it is misplaced, maybe you could push it to the right.